<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PaperDialog</class>
 <widget class="QDialog" name="PaperDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>539</width>
    <height>618</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Stationery</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>70</y>
     <width>491</width>
     <height>491</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="paperTab">
    <attribute name="title">
     <string>Paper</string>
    </attribute>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>451</width>
       <height>19</height>
      </rect>
     </property>
     <property name="text">
      <string>Use the input fields bellow to establish the desired stationery look.</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>50</y>
       <width>441</width>
       <height>19</height>
      </rect>
     </property>
     <property name="text">
      <string>You can later use this stationery in a your printer:</string>
     </property>
    </widget>
    <widget class="QComboBox" name="paperSizeComboBox">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>90</y>
       <width>271</width>
       <height>27</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>90</y>
       <width>101</width>
       <height>19</height>
      </rect>
     </property>
     <property name="text">
      <string>Paper Size</string>
     </property>
    </widget>
    <widget class="QGroupBox" name="orientationGroupBox">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>250</y>
       <width>151</width>
       <height>91</height>
      </rect>
     </property>
     <property name="title">
      <string>Orientation</string>
     </property>
     <widget class="QRadioButton" name="landscapeRadioButton">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>25</y>
        <width>111</width>
        <height>30</height>
       </rect>
      </property>
      <property name="text">
       <string>Landscape</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QRadioButton" name="portraitRadioButton">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>55</y>
        <width>103</width>
        <height>30</height>
       </rect>
      </property>
      <property name="text">
       <string>Portrait</string>
      </property>
     </widget>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>360</y>
       <width>41</width>
       <height>19</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>8</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Inches</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>356</y>
       <width>91</width>
       <height>19</height>
      </rect>
     </property>
     <property name="text">
      <string>Top Margin</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="customSizeW">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>180</y>
       <width>113</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="customSizeH">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>180</y>
       <width>113</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>150</y>
       <width>120</width>
       <height>91</height>
      </rect>
     </property>
     <property name="title">
      <string>Units</string>
     </property>
     <widget class="QRadioButton" name="inchesRadioButton">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>25</y>
        <width>103</width>
        <height>30</height>
       </rect>
      </property>
      <property name="text">
       <string>Inches</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QRadioButton" name="mmRadioButton">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>55</y>
        <width>103</width>
        <height>30</height>
       </rect>
      </property>
      <property name="text">
       <string>Millimeters</string>
      </property>
     </widget>
    </widget>
    <widget class="QLabel" name="orientationLabel">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>260</y>
       <width>82</width>
       <height>82</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTop|Qt::AlignTrailing</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>180</y>
       <width>21</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="text">
      <string>X</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="styleTab">
    <attribute name="title">
     <string>Style</string>
    </attribute>
    <widget class="QCheckBox" name="feedHolesCheckBox">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>19</y>
       <width>151</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>Tractor Feed Holes</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="colorLabel1">
     <property name="geometry">
      <rect>
       <x>25</x>
       <y>140</y>
       <width>141</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Custom Bars Color</string>
     </property>
    </widget>
    <widget class="QLabel" name="colorLabel2">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>140</y>
       <width>141</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>(e.g., #f0fff0)</string>
     </property>
    </widget>
    <widget class="QComboBox" name="barsColorComboBox">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>88</y>
       <width>131</width>
       <height>27</height>
      </rect>
     </property>
    </widget>
    <widget class="QCheckBox" name="decorationCheckBox">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>50</y>
       <width>131</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>Decorated</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="customColor">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>130</y>
       <width>131</width>
       <height>33</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>99</y>
       <width>111</width>
       <height>19</height>
      </rect>
     </property>
     <property name="text">
      <string>Bars Color</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>30</y>
     <width>91</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>Name</string>
   </property>
  </widget>
  <widget class="QPushButton" name="deleteButton">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>20</y>
     <width>81</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Delete</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancelButton">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>570</y>
     <width>88</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
  </widget>
  <widget class="QComboBox" name="stationeryNameComboBox">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>20</y>
     <width>291</width>
     <height>29</height>
    </rect>
   </property>
   <property name="editable">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>tabWidget</tabstop>
  <tabstop>paperSizeComboBox</tabstop>
  <tabstop>inchesRadioButton</tabstop>
  <tabstop>mmRadioButton</tabstop>
  <tabstop>landscapeRadioButton</tabstop>
  <tabstop>portraitRadioButton</tabstop>
  <tabstop>customSizeH</tabstop>
  <tabstop>customSizeW</tabstop>
  <tabstop>cancelButton</tabstop>
  <tabstop>deleteButton</tabstop>
  <tabstop>feedHolesCheckBox</tabstop>
  <tabstop>barsColorComboBox</tabstop>
  <tabstop>decorationCheckBox</tabstop>
  <tabstop>customColor</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
